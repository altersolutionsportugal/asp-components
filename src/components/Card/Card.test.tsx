import React from 'react'
import { render } from '@testing-library/react'

import { Card } from '.'

test('renders component Card', () => {
  const { container } = render(<Card />)
  expect(container.firstChild).toMatchSnapshot()
})
