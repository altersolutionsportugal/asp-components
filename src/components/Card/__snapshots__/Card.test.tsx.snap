// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders component Card should create snapShot 1`] = `
<div
  aria-hidden="true"
>
  <h1>
    Hello
  </h1>
</div>
`;
